modules/ROOT/pages/customize-your-canvas.adoc

```diff
@@ -15,8 +15,7 @@ Visualizer editors can save defined views so that any Visualizer user can access
 When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.
 Layers, Tags and Display Names can all be customized in the Visualizer UI or by using a runtime property.
```

Comment:
Active voice
```suggestion
You can use international letters, numbers, and symbols to customize Layers, Tags, and Display Names values in the Visualizer UI or by using a runtime property.
```

Reply:
Suggest the previous rewording to omit the text that follows.

Reply:
```suggestion
You can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers) to customize Layers, Tags, and Display Names in the Visualizer UI or by using a runtime property.
```